.openshift-ci/crawler/setup-staging.sh

```diff
@@ -1,16 +1,25 @@
 # The below variables ontain a comma delimited list of GCP buckets,
-# Scripts may read from all buckets but only write to the first bucket in the list.
-KERNEL_PACKAGE_STAGING_BUCKET="gs://stackrox-kernel-packages-staging/copy"
-KERNEL_BUNDLE_STAGING_BUCKET="gs://collector-kernel-bundles-public"
-KERNEL_PACKAGE_BUCKET="${KERNEL_PACKAGE_STAGING_BUCKET}"
-KERNEL_BUNDLE_BUCKET="${KERNEL_BUNDLE_STAGING_BUCKET}"
-
-echo "KERNEL_BUNDLE_STAGING_BUCKET=${KERNEL_BUNDLE_STAGING_BUCKET}"
-echo "KERNEL_PACKAGE_STAGING_BUCKET=${KERNEL_PACKAGE_STAGING_BUCKET}"
+# Scripts may read from all buckets but only write to the *first bucket* in the list.
+
+KERNEL_PACKAGE_BUCKET="gs://stackrox-kernel-packages-staging/copy"
```

Comment:
It's a little weird that the `copy/` subdirectory is the staging area for the main builds, but alright.